Source/CombatExtended/CombatExtended/Loadouts/Dialog_ManageLoadouts.cs

```diff
@@ -110,6 +111,8 @@ public Loadout CurrentLoadout
             }
             set
             {
+                if (MP.IsInMultiplayer && _currentLoadout != null)
```

Comment:
The idea was that when the `_currentLoadout` is being changed, we synchronize its name across all players in case it was changed. This won't affect the player who called it, but will change the name for everyone else.

The alternative was to synchronize the name each time a character changes - however, with the way multiplayer handles this it would have to be set on the next tick (from host perspective) in game instead of current one. This would mean that editing the name wouldn't be as smooth of an experience, plus it would get synced once per each character modified - and many synced calls in a very short time can really tank performance.

Reply:
And the assumption is this is never getting set *to* `null`, so that requires no special handling?  That's probably a reasonable assumption, especially given that even the empty/default loadout is not really a `null` loadout.